### PR TITLE
1. Fix the wrong calibration url

### PR DESCRIPTION
**Pull request by JelloHuang**
2. Add module default port
3. Add more about wallet module ,because the state of wallet unlock block the sealer init

### CODE REVIEW

File: docs/guide/Using-venus-Shared-Modules.md [hunk 1]
```diff
@@ -261,6 +261,12 @@ Using process controll like  `systemmd` or `supervisord` is recommended.
 
 :::
 
+:::warning
+
+Pls keep the wallet unlock state.If the state is locked ,it will block sealer "Waiting for confirmation"
```
Comment (Fatman13) [inline docs/guide/Using-venus-Shared-Modules.md:266]: 很好。小的格式问题，英文句号和逗号后面加空格。例，hello, world. hello.

File: docs/modules/README.md [hunk 2]
```diff
@@ -217,3 +217,15 @@ venus sync status
...
+
+```shell
+Auth 8989
+Gateway 45132
+Daemon node 3453
+Miner 12308
+Sealer 2345
+Worker 3456
+```
+If you have some problem with connect to module,pls check the default port whether is opened. We can run telnet to debug.Such as 10.50.110.59 12308
```
Comment (Fatman13) [inline docs/modules/README.md:231]: 同样标点小问题。

**Review by Fatman13 — COMMENTED** [review]
赞👍

**Comment by JelloHuang** [comment]
have updated @Fatman13

**Comment by Fatman13** [comment]
LGTM

**Review by Fatman13 — APPROVED** [review]

**Pull request merged**